src/components/Content/Content.styled.ts

```diff
@@ -34,72 +55,44 @@ export const StyledEditIcon = styled(EditIcon)`
 
 export const ButtonsWrapper = styled.div`
 	align-items: center;
-	display: flex;	
+	display: flex;
 
 	.MuiButton-root {
 		padding-left: 16px;
 	}
 
 	.iconButton {
-		border-right: ${({ isMaxContent }) => isMaxContent ? 0 : `1px solid ${theme.colors.gray200}`};
-		padding-right: ${({ isMaxContent }) => isMaxContent ? 0 : "16px"};
-	}
-`;
-
-export const ContentRow = styled.div`
-	display: flex;
-	width: 100%;
-`;
-
-export const ContentColumn = styled.div`
-	& > div {
-		margin-right: ${({ cols }) => cols === 1 ? 0 : "60px" };
+		border-right: ${({ canShowMore }) => canShowMore ? `1px solid ${theme.colors.gray200}` : 0};
+		padding-right: ${({ canShowMore }) => canShowMore ? "16px" : 0};
 	}
-
-	width: calc(100% / ${pr => pr.cols});
 `;
 
-export const LabelValueWrapper = styled.div`
-	margin-bottom: 16px;
-`;
-
-export const Value = styled.span`
-	color: ${theme.colors.gray600};
-	margin-left: ${({ isFile }) => isFile ? "24px" : "8px"};
-	font-size: ${({ isFile }) => isFile ? "12px" : "14px"};
-`;
-
-// Styles for the paragraph content type
-export const Paragraph = styled.p`
+export const Label = styled.p`
 	color: ${theme.colors.gray700};
 	font-size: 14px;
-	margin: 0;
+	margin-bottom: 8px;
+	margin-top: 0;
 `;
 
-// Styles for the LabelValue content type
-
-export const Label = styled.span`
-	color: ${theme.colors.gray700};
-	font-size: 14px;
+export const FieldContainer = styled.div`
+	margin-bottom: 24px;
 `;
 
-// Styles for the tags content type
+// Transforms styles
 export const ChipsWrapper = styled.div`
-	margin: 24px 0 40px;
+	div {
```

Comment:
I think the chips shouldn't have the 8px, only the label (which I think it already has)